File: src/cs/sqlite_serial.cs
using System;
using System.Globalization;
using System.Data.SQLite;

namespace Calc
{
  class Program
  {
    static void Main(string[] Args)
    {
      SQLiteConnection conn;
      SQLiteCommand cmd;
      string dbPath = $"URI=file:{AppDomain.CurrentDomain.BaseDirectory}../sqlite/database.db";
      int i;
      double x, prob;
      CultureInfo.DefaultThreadCurrentCulture = new CultureInfo("en-US");
      try
      {        
        conn = new SQLiteConnection(dbPath);
        conn.Open();
        cmd = new SQLiteCommand(conn);
        cmd.CommandText = "DROP TABLE IF EXISTS cs_serial; CREATE TABLE cs_serial(id INTEGER PRIMARY KEY " +
        "AUTOINCREMENT, z_score REAL NOT NULL, cumulative_distribution REAL NOT NULL)";
        cmd.ExecuteNonQuery();
        for (i = -500; i <= 500; i++)
        {
          x = (double)i / 100.0;
          prob = Gaussian.GaussianCDF(0, 1, x);
          cmd.CommandText = $"INSERT INTO cs_serial(z_score, cumulative_distribution) VALUES ({x:0.00}, {prob:0.000000});";
          cmd.ExecuteNonQuery();
        }
        conn.Close();
      }
      catch (System.Exception e)
      {
        Console.WriteLine(e.Message);
      }
    }
  }
}

File: src/java/SQLite_serial.java
import java.io.File;
import java.sql.*;
import java.util.Locale;

public class SQLite_serial {
  public static void main(String[] args) {
    Connection conn;
    Statement stmt;
    String dbPath = String.format("jdbc:sqlite:%s/../sqlite/database.db",
        new File(SQLite_serial.class.getProtectionDomain().getCodeSource().getLocation().getPath().toString())
            .getParent());
    int i;
    double x, prob;
    Locale.setDefault(new Locale("en", "US"));
    try {
      conn = DriverManager.getConnection(dbPath);
      stmt = conn.createStatement();
      stmt.executeUpdate("DROP TABLE IF EXISTS java_serial; CREATE TABLE java_serial(id INTEGER PRIMARY KEY " +
          "AUTOINCREMENT, z_score REAL NOT NULL, cumulative_distribution REAL NOT NULL)");
      for (i = -500; i <= 500; i++) {
        x = (double) i / 100.0;
        prob = Gaussian.gaussianCDF(0, 1, x);
        stmt.executeUpdate(
            String.format("INSERT INTO java_serial(z_score, cumulative_distribution) VALUES (%.2f, %f)", x, prob));
      }
    } catch (Exception e) {
      System.err.println(e.getMessage());
    }
  }
}
